cns/cni/initialize.go

```diff
@@ -0,0 +1,36 @@
+package cni
+
+import (
+	"github.com/Azure/azure-container-networking/cni/api"
+	"github.com/Azure/azure-container-networking/cni/client"
+	"github.com/Azure/azure-container-networking/cns"
+	"k8s.io/utils/exec"
+)
+
+// CNICNSInitializer returns an implementation of cns.PodInfoByIPProvider
+// that execs out to the CNI and uses the response to build the PodInfo map.
+func CNICNSInitializer() (cns.PodInfoByIPProvider, error) {
```

Comment:
Can we change the folder name from cns/cni to be something specific (for lack of better word, cnireconciler?